stant-album/UIImage+getAverageColor.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UIImage+getAverageColor.swift
+//  stant-album
+//
+//  Created by Leandro Martins on 02/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
```

Comment:
Why  this file name is UIImage+get... 
This file is an extension, it should indicate it, like UIImageExtension. In future, if needed, it can have more functions.

Reply:
I didn't thought about that. Is a pattern that I've seen in some videos and forums, but it really doesn't makes sense for continuous work. Thank you for the tip.